server/url_translator.go

```diff
@@ -71,14 +74,18 @@ type route struct {
 	validHosts []*regexp.Regexp // If not empty, then the target hostname must be one of these regexes
 }
 
-func parseScheme(targetUrl string) scheme {
+func parseScheme(targetUrl string, header *http.Header) scheme {
 	switch {
 	case targetUrl[0:3] == "ws:":
 		return schemeWS
 	case targetUrl[0:4] == "udp:":
 		return schemeUDP
+	case targetUrl[0:5] == "http:" && (header != nil && header.Get("Accept") == "text/event-stream"):
+		return schemeHTTPSSE
 	case targetUrl[0:5] == "http:":
 		return schemeHTTP
```

Comment:
Please and thank you
```suggestion
	case targetUrl[0:5] == "http:":
		if header != nil && header.Get("Accept") == "text/event-stream" {
			return schemeHTTPSSE
		}
		return schemeHTTP
```

Reply:
Done